DirectProgramming/C++SYCL_FPGA/Tutorials/Features/loop_coalesce/README.md

```diff
@@ -199,7 +199,7 @@ If the innermost coalesced loop has a very small trip count, `loop_coalesce` mig
 ## Read the Reports
 Locate `report.html` in the `loop_coalesce_report.prj/reports/` directory. 
 
-On the main report page, scroll down to the section titled `Compile Estimated Kernel Resource Utilization Summary`. Each kernel name ends in the `loop_coalesce` attribute argument used for that kernel, for example, KernelCompute<2> uses a `loop_coalesce` argument of 2. You can verify that the number of registers, MLABs, and DSPs used for each kernel decreases after nested loops are coalesced.
+On the main report page, scroll down to the section titled `Compile Estimated Kernel Resource Utilization Summary`. Each kernel name ends in the `loop_coalesce` attribute argument used for that kernel; for example, KernelCompute<2> uses a `loop_coalesce` argument of 2. You can verify that the number of ALMs used decreases when loops are coalesced. Since KernelCompute<2> has fewer loops than KernelCompute<1>, it requires less hardware for loop overhead. 
```

Comment:
The "2" in "...uses a `loop_coalesce` argument of 2..." should also be marked with back-ticks because it's a value the user sets and not a quantity.

"...uses a `loop_coalesce` argument of `2`..."

Reply:
"...when loops are...." -> "..when the loops are..."